### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Quintox95** [Matze2010/homebridge-innogyPlatform#1]
Hey Matze,

do you have an Documentation for this, how i can use it in Homebridge?

**Comment by Matze2010** [comment]
Hi, the problem is that innogy has an API that can be used but I did not find a way to get credentials to use this API, yet.

**Comment by Matze2010** [comment]
This API is used by this openHAB plugin: https://www.openhab.org/addons/bindings/innogysmarthome/

**Comment by Quintox95** [comment]
Yes i used OpenHAB, but i cant connect it with HomeKIT anymore (since iOS13). And the homebridge-openhab2-complete Plugin does not work with Innogy Thermostat.

Innogy is very slow and does not share any information what feels really bad for the customers. Do you try to contact Innogy for an dev-account?

**Comment by Matze2010** [comment]
No, it is possible to extract the required credentials by logging the traffic that originates from the app and extracting your credentials from there. I use the iOS app and log the traffic. However you have to setup a ssl proxy, since the connection to the innogy api is encrypted.

**Comment by Quintox95** [comment]
So this Plugin is not finished yet and it will take its time to get in touch with innogy right?

Do you have other Solution at the moment to connect the Innogy Stuff to HomeKit?

**Comment by Matze2010** [comment]
> So this Plugin is not finished yet and it will take its time to get in touch with innogy right?

Right. It is more a proof of concept and testing environment since I have no official documentation of the innogy api
> Do you have other Solution at the moment to connect the Innogy Stuff to HomeKit?
> 
No, sorry
